### PR TITLE
Dependency updates for 3.10.x

### PR DESCRIPTION
**Pull request by Lex-2008**
Update openssl from 1.0.2o to 1.0.2p
Update libyaml from 0.1.7 to 0.2.1
Update libcurl-hub from 7.59.0 to 7.61.1
Update apr from 1.5.2 to 1.6.5
Update apr-util from 1.5.4 to 1.6.1
Update apache from 2.4.33 to 2.4.35
Update git from 2.13.6 to 2.19.1
Update postgresql-hub from 9.6.8 to 9.6.10
Update php from 5.6.35 to 5.6.38
Update redis from 3.2.11 to 3.2.12
Update libcurl from 7.59.0 to 7.61.1

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thank you for submitting a PR! Maybe @craigcomstock can review this?

**Comment by Lex-2008** [comment]
Ready to be reviewed, build is almost green: https://ci.cfengine.com/job/testing-pr/1465/

File: deps-packaging/libattr/no_fancy_gcc.patch [hunk 1]
```diff
@@ -0,0 +1,67 @@
+--- ./include/attributes.h.orig	2018-10-20 11:47:03.719849733 +0200
++++ ./include/attributes.h	2018-10-20 11:57:20.611941178 +0200
+@@ -127,10 +127,10 @@
+  */
+ EXPORT int attr_get (const char *__path, const char *__attrname,
+ 			char *__attrvalue, int *__valuelength, int __flags)
+-	__attribute__ ((deprecated ("Use getxattr or lgetxattr instead")));
++	__attribute__ ((deprecated));
```
Comment (craigcomstock) [inline deps-packaging/libattr/no_fancy_gcc.patch]: Interesting. Why the removal of the deprecated message? Doesn't work on some toolchains?

Comment (Lex-2008) [reply deps-packaging/libattr/no_fancy_gcc.patch]: I believe so. Commit message says:

> Also they do things that require newer GCC than the one available on RHEL/Debian 6.

@vpodzime might give more insights

**Pull request merged**